ert/tracker/evaluator_tracker.py

```diff
@@ -123,7 +125,9 @@ def _drain_monitor(self):
         drainer_logger.debug("tasks complete")
         self._model.teardown_context()
 
-    def track(self):
+    def track(
+        self,
+    ) -> Iterator[Union[FullSnapshotEvent, SnapshotUpdateEvent, EndEvent]]:
```

Comment:
This is actually a generator, which should have the hint structure `Generator[yield_type, send_type, return_type]` so then you get

```python
[Generator[Union[FullSnapshotEvent, SnapshotUpdateEvent, EndEvent], None, None]]
```

See https://docs.python.org/3/library/typing.html#typing.Generator)